PROJECT

```diff
@@ -0,0 +1,28 @@
+domain: openebs.io
+layout: go.kubebuilder.io/v3
+multigroup: true
+plugins:
+  manifests.sdk.operatorframework.io/v2: {}
+  scorecard.sdk.operatorframework.io/v2: {}
+projectName: node-disk-manager
+repo: github.com/openebs/node-disk-manager
+resources:
+- api:
+    crdVersion: v1
+    namespaced: true
+  controller: true
+  domain: openebs.io
+  group: blockdevice
```

Comment:
The group is `openebs.io` right? This will resolve to `blockdevice.blockdevice.openebs.io`